openslides/utils/models.py

```diff
@@ -57,38 +57,40 @@ def get_rest_pk(self):
 
     def save(self, skip_autoupdate=False, information=None, *args, **kwargs):
         """
-        Calls the django save-method and afterwards hits the autoupdate system.
+        Calls Django's save() method and afterwards hits the autoupdate system.
 
         If skip_autoupdate is set to True, then the autoupdate system is not
         informed about the model changed. This also means, that the model cache
-        is not updated.
+        is not updated. You have to do it manually. Like this:
 
-        The optional argument information can be an object that is given to the
-        autoupdate system. It should be a dict.
+        TODO HELP ME
+
+        The optional argument information can be a dictionary that is given to
+        the autoupdate system.
         """
-        # TODO: Fix circular imports
+        #TODO: Add example in docstring.
+        #TODO: Fix circular imports
         from .autoupdate import inform_changed_data
         return_value = super().save(*args, **kwargs)
-        inform_changed_data(self.get_root_rest_element(), information=information)
+        if not skip_autoupdate:
+            inform_changed_data(self.get_root_rest_element(), information=information)
```

Comment:
Do we need to send the root element? The informed_changed_data method already does this.